plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -1328,16 +1332,20 @@ private void alterPartitionWithoutStatistics(HiveIdentity identity, String datab
         }
     }
 
+    // TODO: Today, this method is called exclusively for addPartition or alterPartition.  For transactional tables,
+    // the column statistics will be empty.  Issue #7190 calls for creating stats for transactional INSERT, DELETE and UPDATE.
     private void storePartitionColumnStatistics(HiveIdentity identity, String databaseName, String tableName, String partitionName, PartitionWithStatistics partitionWithStatistics)
     {
         PartitionStatistics statistics = partitionWithStatistics.getStatistics();
         Map<String, HiveColumnStatistics> columnStatistics = statistics.getColumnStatistics();
+        // If the table is a transactional table, the column statistics will be empty for all
+        // operations other than ANALYZE table
```

Comment:
There are also other cases when stats can be empty. For example when `hive.table-statistics-enabled=false`.
I don't think this comment is necessary here.

Reply:
oh, i see it's necessary few lines below, when NO_ACID_TRANSACTION is used.
Maybe move the comment there and reword:

```
// Using NO_ACID_TRANSACTION as this code path is currently not used for transactional tables.
// TODO (https://github.com/trinodb/trino/issues/7190) support stats in INSERT, DELETE and UPDATE for transactional  tables
```

Reply:
Moved and reworded.